README.md

```diff
@@ -210,6 +210,56 @@ helm install my-release marklogic/marklogic --version=1.0.0-ea1 \
 
 We recommend that you use the `values.yaml` file for configuring your installation.
 
+## Cluster Scaling
```

Comment:
I think this section should just be "Adding and Removing Hosts from Clusters". "Scaling" has other connotations that I don't want to confuse this with.

Reply:
Updated.